### PR TITLE
docs for get_duplicates macro v1 (features part)

### PR DESCRIPTION
**Pull request by sergey-vdovin**
get_duplicates macro v1 (features part)

https://github.com/re-data/re-data/issues/143

https://github.com/re-data/dbt-re-data/blob/main/macros/public/filtering/get_duplicates.sql

problems to discuss:
1. separate dbt project for docs entities
2. ability to launch macro on non-dbt table
3. add a description in example project docs
4. descriptions in dbt data catalog https://re-data.github.io/dbt-re-data/#!/macro/macro.re_data.filter_get_duplicates

## What
*Describe what the change is solving*

## How
*Describe the solution*

### CODE REVIEW

File: docs/docs/reference/data_preparation/data_filtering.md [hunk 1]
```diff
@@ -41,7 +42,44 @@ This macro allows you remove duplicate rows from a dbt model based on certain co
...
+```sql
+  id |  status      |   updated_at    |
+--------------------------------------+
+ 1   |  pending     |    13:00:45     |
+ 2   |  completed   |    13:05:23     |
+ 1   |  completed   |    13:10:35     |
+ 2   |  pending     |    13:04:49     |
+ 3   |  completed   |    13:30:00     |
+
+ => SELECT id, status, updated_at,
```
Comment (mateuszklimek) [inline docs/docs/reference/data_preparation/data_filtering.md:69]: as rest docs are using small letters can you make this SELECT/WHERE small?

Comment (sergey-vdovin) [reply docs/docs/reference/data_preparation/data_filtering.md]: done

File: docs/docs/reference/data_preparation/data_filtering.md [hunk 2]
```diff
@@ -41,9 +42,46 @@ This macro allows you remove duplicate rows from a dbt model based on certain co
...
+  id |  status      |   updated_at    |
+--------------------------------------+
+ 1   |  pending     |    13:00:45     |
+ 2   |  completed   |    13:05:23     |
+ 1   |  completed   |    13:10:35     |
+ 2   |  pending     |    13:04:49     |
+ 3   |  completed   |    13:30:00     |
+
+ => select id, status, updated_at,
+	   re_data_duplicates_group_rows_count,
```
Comment (mateuszklimek) [inline docs/docs/reference/data_preparation/data_filtering.md:70]: btw for rows_count, I noticed re_data in other places uses `row_count` so maybe we change in here (in docs) and add small PR regarding this for recently merged changes in dbt-re-data repo? @sergey-vdovin what do you think?

Comment (sergey-vdovin) [reply docs/docs/reference/data_preparation/data_filtering.md:70]: [as i remember we discussed that while in the rest of the word](https://re-data.slack.com/archives/D02J4S82BU2/p1636642008002400

Sergey: Hi ) probably we should use row_count not rows_count like in the rest of dbt and outside of dbt, the same applies to duplicates_count
https://ell.stackexchange.com/questions/162678/row-count-or-rows-count-to-mean-number-of-rows
What do you think about it?)

Mateusz: yea probably, only thing worrying in changing this is what to do with existing metrics which are already generated)

**Pull request merged**